scripts/libs.versions.toml

```diff
@@ -8,10 +12,11 @@ androidx-benchmark-junit4 = "1.1.0-beta04"
 androidx-constraintlayout = "1.0.1"
 androidx-corektx = "1.8.0"
 androidx-lifecycle-compose = "2.5.1"
+androidx-lifecycle-runtime-compose = "2.6.0-alpha02" # merge with above when not alpha
```

Comment:
Is this intentional for this PR?

Reply:
Yes. This PR added it but didn't add it to the global file. https://github.com/android/compose-samples/pull/970